core/server/src/main/java/alluxio/worker/netty/DataServerHandler.java

```diff
@@ -100,134 +80,4 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
     LOG.warn("Exception thrown while processing request", cause);
     ctx.close();
   }
-
-  private void handleBlockReadRequest(final ChannelHandlerContext ctx,
```

Comment:
There's no unit test for this moved logic? Otherwise the tests should be moved, too?